content/applications/hr/recruitment/recruitment_analysis.rst

```diff
@@ -0,0 +1,102 @@
+====================
+Recruitment analysis
+====================
+
+Reporting is a helpful tool for a recruitment department so they can see which job positions have
+the most applicants, which have the most referrals, or how long it takes for applicants to move
+through the pipeline.
+
+Knowing how many applicants each specific job position has, along with the numbers for those hired
+and refused, can provide valuable insights. This information can assist the recruiting team to pivot
+their strategies to acquire the most desirable candidates.
+
+Recruitment analysis report
+===========================
+
+Start by navigating to :menuselection:`Recruitment app --> Reporting --> Recruitment Analysis`.
+This presents a line chart of all applicants for the last year.
+
+Three separate color-coded metrics are presented: :guilabel:`In Progress`, :guilabel:`Hired`, and
+:guilabel:`Refused`.
+
+Hover the cursor over a month of the chart and a pop-up window appears, and displays the specific
+numbers for that month.
+
+.. image:: recruitment_analysis/line-chart.png
+   :align: center
+   :alt: The default Recruitment Analysis report.
+
+Pivot table view
+----------------
+
+For a more detailed view of the information in the :guilabel:`Recruitment Analysis` report, click
+the :icon:`oi-view-pivot` :guilabel:`(Pivot)` icon in the top-right corner. This displays all the
+information in a pivot table.
+
+In this view, the job positions are displayed in the rows, and the total numbers of applicants in
+their various stages, are the columns.
```

Comment:
You should mention that the data displayed in the rows and columns can be changed.